Description: Patch to remove logtail version detection code from the plugin.
 Not fixed upstream.
Origin: other, http://bugs.debian.org/297628
Bug-Debian: http://bugs.debian.org/297628
Forwarded: no
Index: trunk/node/node.d/amavis.in
===================================================================
--- trunk.orig/node/node.d/amavis.in	2008-07-01 19:01:18.000000000 +0200
+++ trunk/node/node.d/amavis.in	2008-07-01 19:02:11.000000000 +0200
@@ -50,23 +50,11 @@
 spamm=U
 spams=U
 
-ARGS=0
-`$LOGTAIL /etc/hosts 2>/dev/null >/dev/null`
-if [ $? = 66 ]; then
-    if [ ! -n "$logtail" ]; then
-	ARGS=1
-    fi
-fi
-
 TEMP_FILE=`mktempfile munin-amavis.XXXXXX`
 
 if [ -n "$TEMP_FILE" -a -f "$TEMP_FILE" ]
 then
-	if [ $ARGS != 0 ]; then
-	    logtail -f ${AMAVIS_LOG} -o $STATEFILE | grep 'amavis\[.*\]:' > ${TEMP_FILE}
-	else
-	    logtail ${AMAVIS_LOG} $STATEFILE | grep 'amavis\[.*\]:' > ${TEMP_FILE}
-	fi
+	logtail ${AMAVIS_LOG} $STATEFILE | grep 'amavis\[.*\]:' > ${TEMP_FILE}
 	total=`cat ${TEMP_FILE} | wc -l`
 	virus=`grep INFECTED ${TEMP_FILE} | wc -l`
 	spamm=`grep 'Passed.*Hits: 1[0-9][.]' ${TEMP_FILE} | wc -l`
